.github/workflows/examples.yml

```diff
@@ -136,10 +136,7 @@ jobs:
           # openmmforcefields incompatible with new toolkit
           NB_ARGS+=" --ignore=examples/external/swap_amber_parameters/swap_existing_ligand_parameters_with_openmmforcefields.ipynb"
 
-          # See https://github.com/openforcefield/openff-toolkit/issues/1392
-          NB_ARGS+=" --ignore=examples/toolkit_showcase"
-
-          python -m pytest $NB_ARGS examples
+          python -m pytest $PYTEST_ARGS $NB_ARGS examples
```

Comment:
Adding `$PYTEST_ARGS` mostly serves the role of counting examples for code coverage. If that's something we want to do.